doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -454,8 +450,7 @@ the latter can be replaced by the open syntax ``of term`` or
 ``(_ : term)`` expression. This feature almost behaves as the
 following extension of the binder syntax:
 
-.. prodn::
-   binder += {| & @term | of @term }
+.. todo .. prodn:: binder += {| & @term | of @term }
```

Comment:
I see these grammar rules are not present in the general term syntax. What is the plan? Their doc is still here, but without the grammar rule it is a bit off.

Reply:
This bit of syntax is in `assumptions.rst` (first page of Core Language/Functions and Assumptions) because it adds to a non-SSR nonterminal.  I added a link from there to here, while the ``` :token:`binder` ``` hyperlinks the other way, to the grammar.  I think the production really should be in the Assumptions chapter to make the grammar browsable.  The production will be very hard to discover with the `binder +=` form here.  Note the "SSR" annotation on the right hand side for the production.

![image](https://user-images.githubusercontent.com/1253341/97257204-73e6ab00-17d2-11eb-842a-2846ec514561.png)

Reply:
I'm fine with that, but I guess we need the last line *also* to appear (in some form) in the SSR doc, otherwise the text talks about a production what is not there.

Also, if possible, I'd rather have 2  lines: `of term10` and `& term10`. Maybe it's just the rendering, but it is not so clear there is a `|` between the two. I'm not 100 sure if the grammar can actually be rewritten as I say, at least in past camlp5 had factoring issues (here it is left factored by hand, which is fine, but does not improve readability IMO).

Reply:
Got it.  `doc_grammar` can handle it with these instructions in `common.edit_mlg`:

```
srbinder: [
| REPLACE [ "of" | "&" ] term99      (* SSR plugin *)
| WITH "of" term99      TAG SSR
| "&" term99      TAG SSR
]
```

I'd expect changing the parser would work, too, but not in this PR.

Reply:
The two forms do exactly the same thing.  Is that right?

```[  ["of" -> { () } | "&" -> { () } ]; c = term LEVEL "99" -> {```

Reply:
Yes, but now that I think about it, any number of & can only follow an of